src/langgraph-platform/egress-metrics-metadata.mdx

```diff
@@ -9,7 +9,7 @@ Self-hosted LangGraph Platform instances store all information locally and will
 
 In the future, we will be introducing support diagnostics to help us ensure that the LangGraph Platform is running at an optimal level within your environment.
 
-<Warning>This will require egress to `https://beacon.langchain.com` from your network. If using an API key, you will also need to allow egress to `https://api.smith.langchain.com` or `https://eu.api.smith.langchain.com` for API key verification.</Warning>
+<Warning>This will require egress to `https://beacon.langchain.com` from your network. If using an API key, you will also need to allow egress to `https://api.smith.langchain.com` or `https://eu.api.smith.langchain.com` for API key verification. Refer to our [whitelisting IP section](/langsmith/cloud-architecture-and-scalability) for static IP addresses, if needed.</Warning>
```

Comment:
```suggestion
<Warning>This will require egress to `https://beacon.langchain.com` from your network. If using an API key, you will also need to allow egress to `https://api.smith.langchain.com` or `https://eu.api.smith.langchain.com` for API key verification. Refer to the [allowlisting IP section](/langsmith/cloud-architecture-and-scalability#allowlisting-ip-addresses) for static IP addresses, if needed.</Warning>
```